app/app/auth.py

```diff
@@ -0,0 +1,41 @@
+from flask import Blueprint, render_template, request, redirect, url_for, flash
+from .models import User
+from werkzeug.security import generate_password_hash, check_password_hash
+from . import db
+from flask_login import login_user, login_required, logout_user, current_user
+
+MIN_USERNAME_LENGTH = 4
+MIN_PASSWORD_LENGTH = 7
+
+auth = Blueprint('auth', __name__)
+
+@auth.route('/login', methods=['GET', 'POST'])
+def admin_login():
+    if current_user.is_authenticated:
+        user = User.query.filter_by(id = current_user.id).first()
+        theme = user.selected_theme()
+    else:
+        theme = "Blue"
```

Comment:
let's keep it MVP first, we can implement that later

Reply:
if anything, this can be saved in the localstorage instead of the database (most common and easiest way to deal with user-preferences data that is small scale) see https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage

Reply:
I have moved this particular feature as it's own task see #31